e2e/package.json

```diff
@@ -35,14 +35,17 @@
     "@reduxjs/toolkit": "1.8.5",
     "@storybook/testing-react": "1.3.0",
     "@testing-library/cypress": "8.0.3",
+    "axe": "^9.0.0",
```

Comment:
we prefer locking the version at a specific number so that upgrades don't potentially add vulnerabilities, just remove the ^ from in front of the version number, it's added by default when you add a new module

Reply:
resolved!